src/ProjectEnvironmentConfiger.ts

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+// The module 'vscode' contains the VS Code extensibility API
+// Import the module and reference it with the alias vscode in your code below
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as utils from './utils';
+
+import {TelemetryContext} from './telemetry';
+import {FileNames, ScaffoldType, PlatformType, TemplateTag} from './constants';
+import {FileUtility} from './FileUtility';
+import {ProjectTemplate, TemplatesType} from './Models/Interfaces/ProjectTemplate';
+import {RemoteExtension} from './Models/RemoteExtension';
+import * as UIUtility from './UIUtility';
+
+const impor = require('impor')(__dirname);
```

Comment:
impor => Please user better naming here.

Reply:
It is the original name of the npm library.